client/cherami/outputhostconnection.go

```diff
@@ -205,6 +205,7 @@ func (conn *outputHostConnection) readMessagesPump() {
 
 		cmd, err := conn.outputHostStream.Read()
 		if err != nil {
+			conn.reporter.IncCounter(metrics.ConsumeMessageFailedRate, nil, 1)
```

Comment:
I don't think ConsumeMessageFailed is very precise here, suggest renaming to OutputStreamReadFailureRate